addons/l10n_ch/report/swissqr_report.xml

```diff
@@ -12,127 +12,92 @@
             <field name="paperformat_id" ref="l10n_ch.paperformat_euro_no_margin"/>
         </record>
 
-        <template id="l10n_ch_swissqr_template">
-            <t t-set="o" t-value="o.with_context(lang=lang)"/>
-            <t t-set="company" t-value="o.company_id"/>
-            <t t-call="web.external_layout_standard">
-                <!-- add default margin for header (matching A4 European margin) -->
-                <t t-set="report_header_style">padding-top:6.2mm; padding-left:8.2mm; padding-right:8.2mm;</t>
+        <record id="l10n_ch_qr_header" model="ir.actions.report">
+            <field name="name">QR-bill Header</field>
+            <field name="model">account.move</field>
+            <field name="report_type">qweb-pdf</field>
+            <field name="report_name">l10n_ch.qr_report_header</field>
+            <field name="report_file">l10n_ch.qr_report_header</field>
+        </record>
+
+        <template id="l10n_ch_header_template">
+            <t t-call="web.external_layout">
+                <!--The 2 following elements are necessary for the header to be displayed correctly.-->
+                <!--(Trust me on that one.)-->
+                <br/>
+                <p>      </p>
```

Comment:
My brain did a little suicide after reading that line :smiling_face_with_tear:

Reply:
Me too. I promise you I'm doing this against my will. :smiling_face_with_tear: